api/client/canary/aws.pb.go

```diff
@@ -32,7 +32,7 @@ type Aws struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// Whether the integration is enabled.
+	// Whether the AWS canary integration is enabled.
```

Comment:
Calling it `AWS Canary` integration is confusing to me.

Really using this allows Kayenta to store canary config and archived results in an S3 bucket.